src/Microsoft.IdentityModel.Clients.ActiveDirectory/Platforms/iOS/WebUI.cs

```diff
@@ -90,5 +119,11 @@ private void CallbackMethod(AuthorizationResult result)
         {
             SetAuthorizationResult(result);
         }
+
+        public void Dispose()
+        {
+            this.didEnterBackgroundNotification.Dispose();
```

Comment:
just in case, can you test and make sure this is called? According to Apple: https://developer.apple.com/documentation/foundation/nsnotificationcenter/1415360-addobserver?language=objc

If your app targets iOS 9.0 and later or macOS 10.11 and later, you don't need to unregister an observer in its dealloc method. Otherwise, **you should call removeObserver:name:object: before observer or any object passed to this method is deallocated.**

Reply:
Verified already. With our our library creating a new WebUI for each authentication call, I was able to verify that this dispose method is called when the user successfully authenticates, authentication fails and when the user cancels the authentication and closes the WebUI. This was tested on a device.